File: data/src/main/java/com/github/gmetal/data/entity/SeasonEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class SeasonEntity {

    @JsonField(name = "air_date")
    public String airDate;

    @JsonField(name = "episode_count")
    public Integer episodeCount;

    @JsonField(name = "id")
    public Integer id;

    @JsonField(name = "poster_path")
    public String posterPath;

    @JsonField(name = "season_number")
    public Integer seasonNumber;
}

File: data/src/main/java/com/github/gmetal/data/entity/SeasonEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class SeasonEntity {

    @SerialName("air_date")
    var airDate: String? = null

    @SerialName("episode_count")
    var episodeCount: Int? = null

    @SerialName("id")
    var id: Int? = null

    @SerialName("poster_path")
    var posterPath: String? = null

    @SerialName("season_number")
    var seasonNumber: Int? = null
}
